scompose/logger/message.py

```diff
@@ -281,6 +281,10 @@ def table(self, rows, col_width=2):
             message = "\t".join(row)
             self.custom(prefix=label, message=message)
 
+    def clear(self):
```

Comment:
I like this!

Reply:
Much cleaner than the [usual pytest way of intercepting stdout](https://docs.pytest.org/en/latest/capture.html), very good idea to have that simple logger!